code/addons/docs/src/plugins/mdx-plugin.ts

```diff
@@ -34,9 +34,11 @@ export async function mdxPlugin(options: Options): Promise<Plugin> {
             '/dist/shims/mdx-react-shim'
           ),
           ...mdxPluginOptions?.mdxCompileOptions,
-          remarkPlugins: [remarkSlug, remarkExternalLinks].concat(
-            mdxPluginOptions?.mdxCompileOptions?.remarkPlugins ?? []
-          ),
+          remarkPlugins: [
+            ...(mdxPluginOptions?.mdxCompileOptions?.remarkPlugins ?? []),
```

Comment:
```suggestion
          rehypePlugins: [
            ...(mdxPluginOptions?.mdxCompileOptions?.rehypePlugins ?? []),
```

I don't think this even works in the current state.